posts/2021-10-01-ga-release-post.adoc

```diff
@@ -0,0 +1,170 @@
+
+For a quick history review of JPMS, please re-visit the *Introduction of the Java Platform Module System (JPMS)* section of our link:https://openliberty.io/blog/2019/02/06/java-11.html[Open Liberty and Java 11 blog].
+
+Starting in Java 16 with link:https://openjdk.java.net/jeps/396[JEP 396], strong encapsulation of the JDK internal elements took another step.  This effectively means the "kill" switch default for the JVM option `--illegal-access` was changed from `permit`, permitting module access violations, to `deny`, denying this access and instead throwing an `IllegalAccessException` when a violation occurs.
+
+Then, in Java 17 with link:https://openjdk.java.net/jeps/403[JEP 403], they took strong encapsulation further.  Now, while the JVM option `--illegal-access` is still recognized as a valid JVM argument, it is effectively hard coded to `deny`, its passed in value is ignored and the following warning message is issued.
+
+```
+OpenJDK 64-Bit Server VM warning: Ignoring option --illegal-access=permit; support was removed in 17.0
+```
+
+Not to worry though.  You still have more time if your applications or dependencies are not completely ready to run in the modularized Java world.  There are some more precise JVM arguments, primarily `--add-opens` and `--add-exports`, that can help bridge the gap until you, and your applications, are fully modular ready.  These arguments allow the individual module exceptions to the JPMS rules that your applications might need.  
+
+For more information on this topic, please review the link:https://docs.oracle.com/en/java/javase/17/migrate/migrating-jdk-8-later-jdk-releases.html#GUID-7BB28E4D-99B3-4078-BDC4-FC24180CE82B[Strong Encapsulation in the JDK] documentation or checkout one of the other helpful links in the link:<Please turn this into a link to the Java 17 resources section>[Java 17 resources] sections below.
+
+== How to run Open Liberty on Java 17
+
+You will need a copy of Java 17.  To try this out, download Java 17 Temurin from link:https://adoptium.net/?variant=openjdk17[adoptium.net].
+
+Then you will need Open Liberty 21.0.0.10, or newer. To run your applications on Open Liberty 21.0.0.10, check out <<run, Run your apps using 21.0.0.10>>.
+
+There are a number of ways to control the Java executable used by the Open Liberty server. The following list describes the order of precedence:
+
+. `JAVA_HOME` in `${server.config.dir}/server.env`
+. `JAVA_HOME` in `${wlp.install.dir}/etc/server.env`
+. `JAVA_HOME` set in the environment
+. searching the `PATH` in the environment
+
+ For more information, please visit our link:{url-prefix}/docs/latest/reference/config/server-configuration-overview.html[server configuration documentation].
+Happy Java 17 experimenting!
```

Comment:
This section is not showing up correctly.  The link is not a link and the whole thing is in a code block, but should not be.